resolwe/flow/processes/archiver.yml

```diff
@@ -0,0 +1,28 @@
+- slug: archiver
+  name: Archive Files
+  data_name: File archive
+  requirements:
+    expression-engine: jinja
+  version: 0.0.1
+  type: "data:archive:"
+  category: other
+  persistence: TEMP
+  description: >
+    Create an archive of output files. The process does not support
+    nested outputs or directory fields.
+  input:
+    - name: data
+      label: Data list
+      type: list:data
+    - name: fields
+      label: Output file fields
+      type: list:basic:string
+  output:
+    - name: archive
+      label: Archive
+      type: basic:file
+  run:
+    language: bash
+    program: |
+      zippp -0 results.zip {% for d in data %}{% for f in fields %}{% if d[f].file %}"{{ d[f].file }}" {% for r in d[f].refs %}"{{r}}" {% endfor %}{% endif %}{% endfor %}{% endfor %}
```

Comment:
Since this is getting quite long, consider using `\` to break things into multiple lines. The code will get more readable and it will be easier to review changes in the future.

Also, surround `r` in `{{r}}` with spaces.